tests/contrib/operators/test_ssh_operator.py

```diff
@@ -70,6 +70,23 @@ def test_hook_created_correctly(self):
         self.assertEqual(TIMEOUT, task.ssh_hook.timeout)
         self.assertEqual(SSH_ID, task.ssh_hook.ssh_conn_id)
 
+    def test_empty_command_is_none(self):
+        from airflow.exceptions import AirflowException
+        TIMEOUT = 20
+        COMMAND = None
+        task = SSHOperator(
+            task_id="test",
+            command=COMMAND,
+            dag=self.dag,
+            timeout=TIMEOUT,
+            ssh_conn_id="ssh_default"
+        )
+        self.assertIsNone(task.command)
+        try:
+            task.execute(None)
+        except AirflowException as e:
```

Comment:
Can you use [self.assertRaisesRegex](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex)? If the exception is not thrown here, then this test will still be successful because no assertion will be checked.

Reply:
@mik-laj the exception **will be** & **should be** thrown when  `command` is empty, which is what we are checking for.

Reply:
@retornam kamil mean that if some change code in `ssh_operator.py` and cause `task.execute(None)` do not raise the error, this test case will pass too. we here and test not only test function, but also avoid regression, WDYT

Reply:
@zhongjiajie  got it. Updated the PR